lib/modules/platform/github/index.md

```diff
@@ -72,6 +72,12 @@ Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterpr
     The installation tokens expire after 1 hour and need to be regenerated regularly.
     Alternatively as environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
 
+<!-- prettier-ignore -->
```

Comment:
The maintainers should probably review the third-party actions? I think we should be careful with our recommendations in the docs.